pkg/ibmvpc/vpc.go

```diff
@@ -284,6 +284,58 @@ func (fip *FloatingIP) IsMultipleVPCs() bool {
 	return false
 }
 
+type ServiceNetworkGateway struct {
```

Comment:
Add a comment to clarify that `ServiceNetworkGateway` is a virtual gateway we add for convenience - it is not a resource that appears in the input configuration file.